app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+  items: {
+    question: "Your role",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].role",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this label to distinguish between different roles"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s experience"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference from an employer may collaborate some degree of information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Organisation",
+  items: {
+    question: "Organisation where you gained experience or volunteered",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].organisation_name",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this label to distinguish between different roles"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Dates of volunteering",
+  items: {
+    question: "Start date
+      End date",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].start_date
+      work_experience.volunteering[].end_date",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Time commitment and responsibilities",
+  items: {
+    question: "Enter details of your time commitment and responsibilities",
+    required: "No",
+    api: {
+      param: "work_experience.volunteering[].description",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "can use this to demonstrate what they learnt during school experience"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Reference may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Role - Role involved working with children",
+  items: {
+    question: "Did this job involve working in a school or with children?",
```

Comment:
Eek, should this say "role" (it's like this on live too)?

Reply:
Made a note to make a card!